src/cf-typography.less

```diff
@@ -48,42 +48,42 @@
 // Color variables
```

Comment:
Same comment as the other PR about having a line up here mentioning what the `$color-` comments are referring to.